.eslintrc.js

```diff
@@ -44,13 +52,50 @@ module.exports = {
     'import/extensions': 'off',
     'import/prefer-default-export': 'off',
 
+    '@typescript-eslint/no-namespace': 'error',
+    // TODO: '@typescript-eslint/no-floating-promises': 'error', // Promises must catch errors or be awaited.
```

Comment:
This is a useful one but should wait until after a release, since it's a bit risky.